### PR TITLE
The Aerostatic Ace

### PR DESCRIPTION
**Pull request by NotMergot**
## About The Pull Request

Adds the rest of the aerostatic outfit to the loadout menu. Follows suit with the aerostatic bomber jacket already available wherein the items are separate from the detective versions.

Already tested, should hopefully pass all the tests.

## Why It's Good For The Game

Why go half way with an amazing looking outfit that hardly anyone gets to see as is? Part of it is already in loadouts for everyone to use, no reason not to add the rest.

## Changelog

:cl:
add: Aerostatic Boots loadout item
add: Aerostatic Gloves loadout item
add: Binoclard Lenses loadout item
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_shoes.dm [hunk 1]
```diff
@@ -2,3 +2,7 @@
 	name = "Elegant Heels"
 	item_path = /obj/item/clothing/shoes/heels/drag/lace
 	ckeywhitelist = list("thedragmeme")
+
+datum/loadout_item/shoes/aerostatic_boots
```
Comment (nevimer) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_shoes.dm:6]:
```suggestion
/datum/loadout_item/shoes/aerostatic_boots
```

Comment (NotMergot) [reply modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_shoes.dm]: knew my headass was gonna forget something simple thanks

File: tgstation.dme [hunk 2]
```diff
@@ -13,6 +13,7 @@
 // END_PREFERENCES
 
 // BEGIN_INCLUDE
+#include "modular_zubbers\code\modules\clothing\outfits\kim.dm"
```
Comment (nevimer) [inline tgstation.dme:16]: don't manually edit this file

File: modular_zubbers/code/modules/clothing/outfits/kim.dm [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/obj/item/clothing/glasses/binoclardlenses
```
Comment (nevimer) [inline modular_zubbers/code/modules/clothing/outfits/kim.dm:1]: underscorethenamesbetweenwords_please

File: modular_zubbers/code/modules/clothing/outfits/kim.dm [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+	name = "Binoclard Lenses"
+	desc = "Shows you know how to sew a lapel and center a back vent."
+	icon = 'modular_zubbers/icons/obj/clothing/kim.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/kim.dmi'
+	icon_state = "binoclard_lenses"
+
+/obj/item/clothing/gloves/aerostaticgloves
```
Comment (nevimer) [inline modular_zubbers/code/modules/clothing/outfits/kim.dm:8]: ditto

File: modular_zubbers/code/modules/clothing/outfits/kim.dm [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	name = "Aerostatic Gloves"
+	desc = "Vivid red gloves that exude a mysterious style, sadly not the best for gardening, or getting bodies out of trees."
+	icon = 'modular_zubbers/icons/obj/clothing/kim.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/kim.dmi'
+	icon_state = "aerostatic_gloves"
+
+/obj/item/clothing/shoes/boots/aerostaticboots
```
Comment (nevimer) [inline modular_zubbers/code/modules/clothing/outfits/kim.dm:15]: ditto

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_glasses.dm [hunk 6]
```diff
@@ -3,7 +3,6 @@
 	item_path = /obj/item/clothing/glasses/trickblindfold/lace
 	ckeywhitelist = list("thedragmeme")
 
-/datum/loadout_item/glasses/surgerygoggles
-	name = "Surgery Goggles"
-	item_path = /obj/item/clothing/glasses/surgerygoggles
-	restricted_roles = list(JOB_MEDICAL_DOCTOR,JOB_CHIEF_MEDICAL_OFFICER)
+/datum/loadout_item/glasses/binoclard_lenses
+	name = "Binoclard Lenses"
+	item_path = /obj/item/clothing/glasses/binoclardlenses
```
Comment (nevimer) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_glasses.dm:8]: same here, has to be consistent

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_gloves.dm [hunk 7]
```diff
@@ -7,3 +7,8 @@
 	name = "Long Rubber Gloves"
 	item_path = /obj/item/clothing/gloves/longrubbergloves
 	restricted_roles = list(JOB_MEDICAL_DOCTOR,JOB_CHIEF_MEDICAL_OFFICER)
+
+/datum/loadout_item/gloves/aerostatic_gloves
+	name = "Aerostatic Gloves"
+	item_path = /obj/item/clothing/gloves/aerostaticgloves
```
Comment (nevimer) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_gloves.dm:13]: and here

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_shoes.dm [hunk 8]
```diff
@@ -2,3 +2,7 @@
 	name = "Elegant Heels"
 	item_path = /obj/item/clothing/shoes/heels/drag/lace
 	ckeywhitelist = list("thedragmeme")
+
+/datum/loadout_item/shoes/aerostatic_boots
+	name = "Aerostatic Boots"
+	item_path = /obj/item/clothing/shoes/boots/aerostaticboots
```
Comment (nevimer) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_shoes.dm:8]: and here

**Review by nevimer — CHANGES_REQUESTED** [review]

File: tgstation.dme (hunk 2 above)
Comment (NotMergot) [reply tgstation.dme:16]: ah right sorry just tick it right?

**Comment by NotMergot** [comment]
GOOD LORD

**Comment by nevimer** [comment]
bap, must be finished up and rebased

**Pull request closed without merging**